rgh-linkify-yolo-issues.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "dom-chef";
+import * as pageDetect from "github-url-detection";
+import { wrap } from "../helpers/dom-utils";
+import features from "../feature-manager";
+import featureLink from "../helpers/feature-link";
+import { getNewFeatureName } from "../options-storage";
+import { isAnyRefinedGitHubRepo } from "../github-helpers";
+import observe from "../helpers/selector-observer";
+function init(): void {
+	for (const issueCell of select.all("table-selector td:nth-child(2)")) {
```

Comment:
What's `table-selector`? This PR hasn't been tested. Test before marking as "ready"